tests/saitests/py3/sai_qos_tests.py

```diff
@@ -1210,9 +1210,19 @@ def runTest(self):
                                 assert (pg_cntrs[i] == pg_cntrs_base[i])
                     else:
                         if i == pg:
-                            assert (pg_cntrs[pg] == pg_cntrs_base[pg] + len(dscps))
+                            if i == 0 or i == 4:
+                                assert (pg_cntrs[pg] >=
+                                        pg_cntrs_base[pg] + len(dscps))
+                            else:
+                                assert (pg_cntrs[pg] ==
+                                        pg_cntrs_base[pg] + len(dscps))
                         else:
-                            assert (pg_cntrs[i] == pg_cntrs_base[i])
+                            # LACP packets are mapped to queue0 and tcp syn packets for BGP to queue4
+                            # So for those queues the count could be more
+                            if i == 0 or i == 4:
```

Comment:
Hi @JibinBao , why BGP packets are mapped to queue 4?

Reply:
> Hi @JibinBao , why BGP packets are mapped to queue 4?

hi @bingwang-ms ,
This is the orignal code, I just revert it back. Please see the link:https://github.com/sonic-net/sonic-mgmt/pull/9859/files